packages/terra-list/src/MultiSelectList.jsx

```diff
@@ -163,16 +163,10 @@ class MultiSelectList extends React.Component {
       newProps.isSelected = isSelected;
     }
 
```

Comment:
Looks like it was removed in a previous commit, but if the item isn't selectable, I don't believe we should be wrapping its onClick or KeyDown as we have no interaction with the element.